src/data-pipeline/etl-common/src/main/resources/ts/traffic_source_channel_rule_v0.json

```diff
@@ -0,0 +1,494 @@
+[{
```

Comment:
Just keep in mind that this ruleset is dynamic, customer could have more rules or less rules

Reply:
yes. this is default one. new rule files will be updated to s3 and submit to EMR job in a lambda when start a new EMR job.